tests/test_class_to_config.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, division, print_function
+
+import environ
+
+
+@environ.config(prefix="APP")
+class AppConfig(object):
+    host = environ.var("127.0.0.1")
+    port = environ.var(5000, converter=int)
+
+
+def test_default():
+    """Class based `from_environ` without `environ` argument.
```

Comment:
For the future, this is not my/our docstrig style (it looks squenched, doesn't it?) but I'll quickfix it myself – thanks!